docs/topic-guides/blendable-web-apps/web-components.md

```diff
@@ -10,48 +10,43 @@ Web Components are a set of new features in HTML 5 and beyond. Each feature is u
-
-The `<template>` tag that was added to HTML5 as part of the Web Components spec family. It allows to define reusable chunks of HTML that work with any framework that supports this new web standard.
+The `<template>` tag that was added to HTML5 to define reusable chunks of HTML that work with any framework.
 
 ### Custom Elements and Shadow DOM
 
 We suggest to keep the business and application logic on the server. Yet, there is still plenty of room to use the capabilities of a modern web browser to run the _view_ logic.
 
-Custom Elements and Shadow DOM define how one can wrap a complex view functionality into a simple HTML tag, that requires no knowledge about the implementation details from the developer.
+Custom elements and Shadow DOM define how one can wrap a complex view functionality into a simple HTML tag, that requires no knowledge about the implementation details from the developer.
 
-As can be seen in the above code snippet, to render a UPC-A barcode using SVG in a web browser, you can just import and use a MIT-licensed Custom Element called [x-barcode](https://github.com/girliemac/x-barcode). There is no learning curve and you don't need to care about the implementation details.
+As can be seen in the above code snippet, to render a UPC-A barcode using SVG in a web browser, you can just import and use a MIT-licensed custom element called [x-barcode](https://github.com/girliemac/x-barcode). There is no learning curve and you don't need to care about the implementation details.
 
-Thousands of open sourced Custom Elements can be found on [customelements.io](https://customelements.io).
+Thousands of open sourced custom elements can be found on [webcomponents.org](https://webcomponents.org).
 
 ### HTML Imports
 
-Last but not least, the HTML Imports spec is the part of Web Components family that defines how to obtain fragments of code using a `<link>` tag. As can be seen on the above snippet, this is how a definition of a Custom Element is loaded.
+Last but not least, the HTML Imports spec is the part of Web Components family that defines how to obtain fragments of code using a `<link>` tag. As can be seen on the above snippet, this is how a definition of a custom element is loaded.
 
-In addition to loading Custom Elements, Starcounter takes benefit from HTML Imports to compose complex apps using small templates known as [partials](html-views.md), but you don't need to care much about that because it is done behind the scenes.
+In addition to loading custom elements, Starcounter takes benefit from HTML Imports to compose complex apps using small [views](html-views.md), but you don't need to care much about that because it is done behind the scenes.
 
 ## Using Web Components
 
 Web Components are loaded by browser with HTML Imports. HTML Import for Polymer in Starcounter should look like this:
 
-```markup
```

Comment:
Are you sure it should be `html` and not `markup`?

Reply:
I am not sure, there is no info about it on https://betadocs.gitbook.com/features/markdown :(

`html` is what I always used in GitHub and Wordpress. But I agree I should keep what was there. I will revert this change.